component/portal/src/main/resources/db/changelog/portal-rdbms.db.changelog-1.0.0.xml

```diff
@@ -349,4 +349,16 @@
     </addColumn>
   </changeSet>
 
+  <changeSet author="portal" id="1.0.0-32">
+    <addColumn tableName="PORTAL_PAGES">
+      <column name="PAGE_TYPE" type="INT" defaultValueNumeric="0" >
+        <constraints nullable="false" />
+      </column>
+      <column name="LINK" type="TEXT" />
+    </addColumn>
+    <addColumn tableName="PORTAL_NAVIGATION_NODES">
+      <column name="TARGET" type="INT" defaultValueNumeric="0" />
+      <column name="DESCRIPTION" type="TEXT" />
```

Comment:
description is not yet added as field in frontend, so IMO to be defined here later